package.json

```diff
@@ -160,19 +161,21 @@
     "gen:ts": "openapi-generator-cli generate --generator-key ts"
   },
   "dependencies": {
+    "cp-cli": "^2.0.0",
     "fetch-ponyfill": "^7.1.0",
     "yargs": "^17.3.1"
```

Comment:
Remove? I don't think we use `yargs` for anything.

Reply:
done